tests/utils/decorators/test_converters.py

```diff
@@ -74,20 +74,25 @@ def test_angular_freq_to_hz_preserves_signature() -> None:
     """
 
     @angular_freq_to_hz
-    def test_func(pos_only, /, arg, *args, required_kwarg, optional_kwarg=2, **kwargs):
+    def test_func(
+        pos_only, /, arg, *args, required_kwarg, optional_kwarg: int = 2, **kwargs
+    ):
         return 2 * u.rad / u.s
 
-    original_signature = inspect.signature(test_func)
-    expected_signature = inspect.signature(
-        lambda pos_only,
+    def func_with_expected_signature(
+        pos_only,
         /,
         arg,
         *args,
         required_kwarg,
-        optional_kwarg=2,
-        to_hz=False,
-        **kwargs: None
-    )
+        optional_kwarg: int = 2,
+        to_hz=False,  # adding type hint here will break signature comparison below
```

Comment:
🌠 Unfortunately, there doesn't appear to be a command line option for `autotyping` to tell it to exclude a file.  I'll try to remember to raise an issue in the `autotyping` repo.